mcmc/FitterBase.h

```diff
@@ -64,9 +76,15 @@ class FitterBase {
   /// @brief Save the settings that the MCMC was run with.
   void SaveSettings();
 
+  /// @brief Used by sigma variation, check how 1 sigma changes spectra
+  inline TGraphAsymmErrors* MakeAssymGraph(TH1D* sigmaArrayLeft, TH1D* sigmaArrayCentr, TH1D* sigmaArrayRight, std::string title);
```

Comment:
"MakeAssymGraph" I think should be "MakeAsymGraph" i.e. "MakeAss" needs to go to "MakeAs"